baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/LogsafeArgName.java

```diff
@@ -54,6 +55,7 @@ public LogsafeArgName() {
         this.unsafeParamNames = ImmutableSet.of();
     }
 
+    @Inject
     public LogsafeArgName(ErrorProneFlags flags) {
```

Comment:
Caught by [`InjectOnBugCheckers`](https://errorprone.info/bugpattern/InjectOnBugCheckers) in [2.18.0](https://github.com/google/error-prone/releases/tag/v2.18.0).